src/cryptoadvance/specter/server_endpoints/wallets/wallets.py

```diff
@@ -894,3 +894,22 @@ def settings(wallet_alias):
         error=error,
         scroll_to_rescan_blockchain=scroll_to_rescan_blockchain,
     )
+
+
+@wallets_endpoint.route("/wallet/<wallet_alias>/settings/clearcache", methods=["POST"])
+@login_required
+def settings_clearcache(wallet_alias):
+    wallet: Wallet = app.specter.wallet_manager.get_by_alias(wallet_alias)
+    error = None
+    wallet.clear_cache()
+    flash("Cache cleared successfully!")
```

Comment:
Not so clear to the user which cache this is referring to.

Reply:
The tooltip looks like this:
![image](https://user-images.githubusercontent.com/117085/211579460-0585b304-b801-498b-a850-dcfdb7b7e631.png)